pandas/core/dtypes/common.py

```diff
@@ -220,6 +220,10 @@ def is_categorical(arr):
     return isinstance(arr, ABCCategorical) or is_categorical_dtype(arr)
 
 
+def is_range(arr):
```

Comment:
this is not similar to the other methods, which detect a type. a RangeIndex is not a type.